test/cli/commands_test.py

```diff
@@ -1300,22 +1348,3 @@ def test_cli_lint_enabled_progress_bar_multiple_files(
         assert r"\rlint by rules:" in raw_output
         assert r"\rrule L001:" in raw_output
         assert r"\rrule L049:" in raw_output
-
-    def test_cli_lint_disabled_progress_bar_when_verbose_mode(
```

Comment:
This test has issues that I couldn't figure out, even in the `main` branch. More notes below.